src/core/server/metrics/collectors/types.ts

```diff
@@ -19,6 +20,10 @@ export interface MetricsCollector<T> {
  * @public
  */
 export interface OpsProcessMetrics {
+  /** pid of the kibana process */
+  pid: number;
+  /** name of process (example: 'coordinator' | 'server_worker' | 'task_worker' | 'reporting_worker') */
+  name: string;
```

Comment:
nit: we should probably use a union string type here with the currently supported values
```suggestion
  name: 'coordinator' | 'server_worker';
```